tests/test_02_04_model.py

```diff
@@ -121,6 +121,7 @@ def get_result(self):
                 'omschrijvingBurgerlijkeStaat': 'Gehuwd',
                 'omschrijvingGeslachtsaanduiding': 'Man',
                 'omschrijvingIndicatieGeheim': 'Geen beperking',
+                'indicatieGeheim': False,
```

Comment:
Je hebt ook `omschrijvingIndicatieGeheim` al

Reply:
Yeah maar die wordt sowieso gevuld. met 'Geen beperking' als dat de default is kan hij weg maar liever gebruiken we de boolean.